src/utils/readers/documents.py

```
@@ -124,6 +125,20 @@ def read_pdf_file(file_path: str | Path, max_pages: int = 5) -> str:
         raise FileReadError(f"Failed to read PDF file {file_path}: {e}") from e
 
 
+def read_rtf_file(file_path: str | Path, max_chars: int = 50_000, **kwargs: Any) -> str:
```

Ruff (0.15.10)

[warning] Unused function argument: `kwargs`

(ARG001)